make0x.py

```diff
@@ -9,11 +9,16 @@
 def output_on_failure(cmd, cwd):
     print(f'$ cd {cwd} && {cmd}')
     ret = subprocess.run(
-        cmd, stdout=subprocess.PIPE, stderr=subprocess.STDOUT, cwd=cwd,
+        cmd, cwd=cwd, stdout=subprocess.PIPE, stderr=subprocess.STDOUT,
     )
     if ret.returncode:
+        try:
+            stdout = ret.stdout.decode()
+        except UnicodeError:
+            stdout = ret.stdout
+
         raise AssertionError('Command {!r} returned {}!\nOutput:\n{}'.format(
-            cmd, ret.returncode, ret.stdout,
+            cmd, ret.returncode, stdout,
```

Comment:
This is my last bet :) But if you don't like it, I will revert.

Reply:
The `CalledProcessError` exception also has the same problem with the `stdout` bytes attribute.